test/models/event_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class EventTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
please add model tests
https://guides.rubyonrails.org/testing.html#rails-meets-minitest

Reply:
We're going to add the model test in the next issue, thanks for the observation.